awx/ui/client/src/notifications/notificationTemplates.form.js

```diff
@@ -581,7 +581,93 @@ export default ['i18n', function(i18n) {
                 ngShow: "notification_type.value == 'slack' ",
                 ngDisabled: '!(notification_template.summary_fields.user_capabilities.edit || canAdd)',
                 awPopOver: i18n._('Specify a notification color. Acceptable colors are hex color code (example: #3af or #789abc) .')
-            }
+            },
+            customize_messages: {
+              label: i18n._('Customize messages…'),
+              type: 'toggleSwitch',
+              toggleSource: 'customize_messages',
+              class: 'Form-formGroup--fullWidth',
+              ngDisabled: '!(notification_template.summary_fields.user_capabilities.edit || canAdd)',
+            },
+            custom_message_description: {
+                type: 'alertblock',
+                ngShow: "customize_messages",
+                alertTxt: i18n._('Use custom messages to change the content of notifications ' +
+                    'sent when a job starts, succeeds, or fails. Use curly braces to access ' +
+                    'information about the job: <code ng-non-bindable>{{ job_friendly_name }}</code>, ' +
```

Comment:
I thought we were going to pull this sort of help text from the OPTIONS response?